core/domain/question_services_test.py

```diff
@@ -99,7 +99,13 @@ def test_delete_question(self):
             1, 'col1', 'en')
 
         question_id = question_services.add_question(self.owner_id, question)
-        question_services.delete_question(self.owner_id, question_id)
+        with self.assertRaisesRegexp(Exception, (
+            'The question with ID %s is not present in collection with ID random' % # pylint: disable=line-too-long
+            (question_id))):
+            question_services.delete_question(self.owner_id, 'random', (
```

Comment:
It is cleaner to read this if you break after `delete_question(`. Ditto below in multiple places. That way, you also avoid introducing an unnecessary set of parens.

Reply:
updated.